editor/filesystem_dock.cpp

```diff
@@ -3893,7 +3899,7 @@ FileSystemDock::FileSystemDock() {
 	folder_colors["pink"] = Color(1.0, 0.271, 0.588); // TTR("Pink")
 	folder_colors["gray"] = Color(0.616, 0.616, 0.616); // TTR("Gray")
 
-	assigned_folder_colors = ProjectSettings::get_singleton()->get_setting("file_customization/folder_colors");
+	assigned_folder_colors = GLOBAL_GET("file_customization/folder_colors");
```

Comment:
Still getting the warning here.

Reply:
`GLOBAL_GET` and `ProjectSettings::get_singleton()->get_setting` seem to be different. The latter has a default fallback, the former not.